src/main/java/org/moltimate/moltimatebackend/util/ProteinUtils.java

```diff
@@ -45,14 +49,15 @@ public static Structure queryPdb(String pdbId) {
             try {
                 return MMCIF_FILE_READER.getStructureById(pdbId);
             } catch (IOException mmcifReaderError) {
-                mmcifReaderError.printStackTrace();
+//                mmcifReaderError.printStackTrace();
```

Comment:
Do we need this?

Reply:
I commented out the `printStackTrace()` instead of removing it. I'm in favor of removing it since we know that the error is being caused because a bad PDB id being requested.

I believe @steplica put it in, I just haven't asked him for his opinion on it yet